rules-tests/TypeDeclaration/Rector/ClassMethod/BoolReturnTypeFromStrictScalarReturnsRector/Fixture/skip_construct.php.inc

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\BoolReturnTypeFromStrictScalarReturnsRector\Fixture;
+
+final class SkipConstructor
+{
+    public function __construct()
+    {
+        return true;
```

Comment:
I am aware that having a method returning something in `__construct` does not actually make sense, but I stumbled over such a case today, and rector was adding a `:bool` return type on said method - which leads to a fatal error at runtime